data.sql

```diff
@@ -12,3 +14,16 @@ INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUE
 INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Blossom','1998-10-13',3,true,17);
 INSERT INTO animals(name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES('Ditto','2022-05-14',4,true,22.0);
 
+-- Owners Table
+INSERT INTO owners(full_name,age) VALUES('Sam Smith',34);
+INSERT INTO owners(full_name,age) VALUES('Jennifer Orwell',19);
+INSERT INTO owners(full_name,age) VALUES('Bob',45);
+INSERT INTO owners(full_name,age) VALUES('Melody Pond',77);
+INSERT INTO owners(full_name,age) VALUES('Dean Winchester',14);
+INSERT INTO owners(full_name,age) VALUES('Jodie Whittaker',38);
+
+-- Species Table
+
+INSERT INTO species(name) VALUES('Pokemon');
+INSERT INTO species(name) VALUES('Digimon');
+
```

Comment:
- You have done great so far :+1: :+1: But there are a few Data you missed inserting into the DB. :+1: :+1: 

> Modify your inserted animals so it includes the species_id value:
If the name ends in "mon" it will be Digimon
All other animals are Pokemon
Modify your inserted animals to include owner information (owner_id):
Sam Smith owns Agumon.
Jennifer Orwell owns Gabumon and Pikachu.
Bob owns Devimon and Plantmon.
Melody Pond owns Charmander, Squirtle, and Blossom.
Dean Winchester owns Angemon and Boarmon.

- [ ] You didn't write the query for this requirement above, and it's definitely going to affect the result of your queries. I'll suggest you add the data and re-run your queries. :+1: :100: